src/utils/command_interface.py

```diff
@@ -159,7 +159,16 @@ def command_status(self, discord_id: str) -> str:
             if not challenge:
                 result += "No current challenges"
             else:
-                result += f"User {leetcode_user_id}'s Weekly Challenge status:\n"
+                questions = self.database.table_weeklyquestion_load_by_challenge_id()
+                total = len(questions)
+                completed = 0
+                leetcode_id = user["leetcode_id"]
+                for q in questions:
+                    if LeetcodeUtil.check_challenge_completion(leetcode_id, q["title_slug"]):
+                        completed += 1
+                percentage = completed/total
+                result += f"User {leetcode_user_id}'s Weekly Challenge status: {percentage}% \
```

Comment:
See if you can normalize the percentage. The way you have it is going to be some decimal between 0 and 1. We want between 0% and 100% as whole numbers. Do something like `int(percentage * 100)`

Reply:
percentage = int(completed/total) * 100

How about this?

Reply:
No, that is incorrect. `completed` will always be less than or equal to `total`. Therefore, `completed/total` will always be a decimal number between 0.0 and 1.0. If it is anywhere in between, `int(completed/total)` will cast a 0.something decimal into a flat out 0. You need to multiply the result by 100 first, then cast to an int.